pages/en/lb3/Model-definition-JSON-file.md

```diff
@@ -575,9 +584,34 @@ For example, to map a property to a column in an Oracle database table, use the
 
 ## Exclude properties from base model
 
-By default, a model inherits all properties from the base. To exclude some base properties from being visible, you need to set the property to `false` or `null`.
+By default, a model inherits all properties from the base. To exclude some base properties from being visible, you need to set excludeBaseProperties = []
```

Comment:
Maybe `set excludeBaseProperties = ['property-be-excluded-from-base-model']`.

I guess we should not remove **set the property to `false` or `null`**. We should state that `excludeBaseProperties` is now the recommended way.